text/0124-documents-api.md

```diff
@@ -20,21 +20,25 @@ Documents are stored inside indexes.
 
 Manipulate documents of a Meilisearch index.
 
-- [3.1.1. `GET` - `indexes/:index_uid/documents`](#311-get---indexesindexuiddocuments)
+- [3.1.1. `GET` - `indexes/:index_uid/documents`](#311-get-indexesindexuiddocuments-and-post-indexesindexuiddocumentsfetch)
```

Comment:
Can we separate `GET - indexes/:index_uid/documents` and `POST - indexes/:index_uid/documents/fetch`? Both are mixed under the same title described by the HTTP verb `GET`, which is inaccurate

Reply:
Actually, since both routes serve the exact same purpose with the same parameter and error code, I wanted to do something similar to what we do with [the search route](https://github.com/meilisearch/specifications/blob/main/text/0118-search-api.md#3-functional-specification) and group both routes in the title.
Don't you think it would be easier to read the spec if I just updated the title? :thinking:

Reply:
What do you think of grouping both routes under a common « human-friendly » name like that; https://github.com/meilisearch/specifications/pull/234/commits/9af7b723789988aac793df988bac3b353e8cb665

Reply:
How would you call it, then?